.vortex/cli/src/Process/Processor.php

```
@@ -0,0 +1,192 @@
+   *   The collected answers.
+   * @param \DrevOps\PhpTui\Handler\HandlerRegistry $handlers
+   *   The handler registry resolving a question id to its handler class.
+   * @param \DrevOps\VortexCli\Utils\Config $config
+   *   The CLI configuration the handlers operate on.
+   * @param array<string,int> $weights
+   *   The processing weight of each question id; its keys are every question.
+   *
+   * @return array<string,mixed>
+   *   The answers, keyed by every question id.
+   */
+  public function responses(Answers $answers, HandlerRegistry $handlers, Config $config, array $weights): array {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$handlers'. (undefined)

(UnusedFormalParameter)